packages/commerce-sdk-react/src/hooks/utils.ts

```diff
@@ -10,12 +10,20 @@ import {ShopperCustomersMutationType} from './ShopperCustomers'
 import {ShopperOrdersMutationType} from './ShopperOrders'
 import {ShopperBasketsMutationType} from './ShopperBaskets'
 
+export type QueryFnData =
+    | DataType<Client[ShopperBasketsMutationType]>
+    | undefined
+    | ((
+          data: DataType<Client[ShopperBasketsMutationType]>
```

Comment:
The react query doc's show that the updater function signature as `updater: TData | (oldData: TData | undefined) => TData`, does our definition conform to that? Specifically the `undefined` part.

Reply:
It does not. Good call on this one. We are currently using updater as a function for all cases. I've updated the change to reflect that